Assets/Scripts/Whale/WhaleAI.cs

```diff
@@ -9,7 +9,7 @@ public class WhaleAI : MonoBehaviour
     private Transform[] RailPoints;
     private AIShip flightController;
 
-    private int curPoint = 0;
+    public int curPoint = 0;

```

Comment:
I think I'd prefer the pattern of keeping this private but adding a public member for `InitialPoint`